src/components/PokemonTable/PokemonTableContainer.js

```diff
@@ -5,19 +5,19 @@ import pokemonTableStyles from "./PokemonTable.style";
 import { HEADER_CELLS } from "../../constants";
 import PokemonTable from "./PokemonTable";
 import store from "../../redux/store";
-import {fetchPokemon} from "../../redux/pokemonSlice";
-import {fetchPokemonById} from "../../redux/selectedPokemonSlice";
+import { fetchPokemon } from "../../redux/pokemonSlice";
+import { fetchPokemonById } from "../../redux/selectedPokemonSlice";
 
 export default function PokemonTableContainer() {
   const { pokemonTypes } = useSelector((state) => state);
   const { pokemon, total } = useSelector((state) => state.pokemon);
   const [data, setData] = useState({
```

Comment:
Probably it's not the best idea to create only one state for the component, cause it contains different types of data. Instead, you can separate this state into small groups by for example functionality like you can group 'pageSize' and 'pageIndex' in one state, but other types of data will be in different states.

Reply:
Done